### ISSUE TITLE
Instructions for building ARM64 images stopped working on Apple Silicon

### ISSUE THREAD

**Issue by jhanggi** [neo4j/docker-neo4j#326]
I was previously able to build working images for my M1 mac using the steps in [DEVELOPMENT.md](https://github.com/neo4j/docker-neo4j/blob/master/DEVELOPMENT.md#building-arm64-based-images). I recently tried to upgrade to a newer neo4j version and discovered the results of the build no longer work.

I checked the history of `make-arm64.mk` and it appears to be the ["makefiles now compliant with docker requirements for arm images"](https://github.com/neo4j/docker-neo4j/commit/9e86b732c0d56bda54203ad477cdaf4d59e8a1b7#diff-f94e3eab322f235ee9b745369540e8b3edd4570a762c26ab7b04773aa4a6295c) commit that broke it.

- **Steps to reproduce**.
```bash
git checkout master
git pull
NEO4JVERSION=4.3.8 gmake -f make-arm64.mk clean tag-arm
docker run --publish=7474:7474 --publish=7687:7687 --volume=neo4j_data:/data --volume=neo4j_plugins:/plugins neo4j:4.3.8                              
> qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory

# Checkout one commit before "makefiles now compliant with docker requirements for arm images"
git checkout 9e86b732c0d56bda54203ad477cdaf4d59e8a1b7~1
NEO4JVERSION=4.3.8 gmake -f make-arm64.mk clean tag-arm
docker run --publish=7474:7474 --publish=7687:7687 --volume=neo4j_data:/data --volume=neo4j_plugins:/plugins neo4j:4.3.8                              
# starts just fine
```

- Expected behaviour

I expect the container to start

- Actual behaviour

When trying to start the container, I get this error: `qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory`

- Neo4j image tag being used, eg `neo4j:latest`, `neo4j:enterprise-3.5` etc

I've tested several tags, in 4.2 and 4.3 including both community and enterprise

- The output of the `docker version` command
```
Client:
 Cloud integration: v1.0.22
 Version:           20.10.11
 API version:       1.41
 Go version:        go1.16.10
 Git commit:        dea9396
 Built:             Thu Nov 18 00:36:09 2021
 OS/Arch:           darwin/arm64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.11
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.9
  Git commit:       847da18
  Built:            Thu Nov 18 00:34:44 2021
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.4.12
  GitCommit:        7b11cfaabd73bb80907dd23182b9347b4245eb5d
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```
- Operating system: (for example Windows 95/Ubuntu 16.04)
macOS Big Sur versions 11.6.1
Chip: Apple M1

**Comment by hserranome** [comment]
Thanks for the info. As a workaround, doing `git checkout 9b93f7507acd0eda8f1fcf7fd8904a8b60525a23` allows me to use the version previous to this issue

**Comment by jennyowen** [comment]
I merged pr #330 so this should work for you again now, but do let me know if it's still broken.

**Comment by jhanggi** [comment]
Looks to be working. I built the latest of the previous versions 4.2.14 and 4.3.10 and they're working great. Thanks for the fix!
